src/lib.rs

```diff
@@ -195,9 +197,11 @@ impl PyVM {
 
     #[pyo3(signature = (error, description=None))]
     fn notify_error(mut self_: PyRefMut<'_, Self>, error: String, description: Option<String>) {
-        self_.vm.notify_error(
+        CoreVM::notify_error(
+            &mut self_.vm,
             Cow::Owned(error),
             description.map(Cow::Owned).unwrap_or(Cow::Borrowed("")),
+            None, // TODO: use the next retry hint
```

Comment:
You don't need to use this. I put it here as in case we want to let people manually create custom retry policies. If you wanna use the default retry policy mechanism (which is what i do in the rust sdk), just pass the retry policy in sys_run_exit

Reply:
yes exactly remains `None`

Reply:
In rust SDK all I do to use the retry policy is the following:

* Let the user provide a retry policy configuration: https://github.com/restatedev/sdk-rust/blob/main/src/context/run.rs#L34
* Pass it to `sys_run_exit` like this: https://github.com/restatedev/sdk-rust/blob/main/src/endpoint/context.rs#L738

Reply:
The other change you need is that now, for both retryable and terminal error, you need to call `sys_run_exit` (that's the reason for `RunExitResult`), so that internally the vm can decide whether to convert the retryable error in terminal error, or just expose the terminal error.